src/parser/jobs/rdm/modules/Swiftcast.tsx

```diff
@@ -0,0 +1,31 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Action} from 'data/ACTIONS'
+import {BASE_GCD} from 'data/CONSTANTS'
+import {dependency} from 'parser/core/Injectable'
+import Downtime from 'parser/core/modules/Downtime'
+import {Swiftcast as CoreSwiftcast} from 'parser/core/modules/Swiftcast'
+import React from 'react'
+import {DISPLAY_ORDER} from './DISPLAY_ORDER'
+
+export class Swiftcast extends CoreSwiftcast {
+static override displayOrder = DISPLAY_ORDER.SWIFTCAST
+
+@dependency private downtime!: Downtime
+
+override considerSwiftAction(_action: Action): boolean {
+	//We want to inspect the cast time to determine if the player was allowed to use SwiftCast or not
+	const castTime = _action?.castTime ?? 0
+	if (castTime > 0) {
+		//As long as we aren't in downtime, the cast time must exceed the base GCD to qualify.
+		if (!this.downtime.isDowntime() && castTime <= BASE_GCD) {
+			return false
+		}
+		//Then it had to be VerRaise, VerAero III, or VerThunder III or we were in downtime so it's valid
+		return true
+	}
+	//Use the default behavoir if we've gotten back no cast time
+	return super.considerSwiftAction(_action)
+}
```

Comment:
Looks like the indentation on these got a bit janky, could you clean these up?

Also, the `_action` variable name in the base class is because the variable doesn't get used in the base implementation -- when overriding, you should go ahead and remove the underscore from the variable name and just use `action`

Reply:
Oh I didn't realize you could change the textual name when overriding it like that.  Good to know.